velox/docs/develop/operators.rst

```diff
@@ -570,6 +570,8 @@ If no sorting columns are specified then the order of the results is unspecified
     - Output column names for each window function invocation in windowFunctions list below.
   * - windowFunctions
     - Window function calls with the frame clause. e.g row_number(), first_value(name) between range 10 preceding and current row. The default frame is between range unbounded preceding and current row.
+  * - isStreamingWindow
```

Comment:
Perhaps, rename to inputsSorted to clarify that inputs are clustered on partitioned keys and then sorted by sorting keys using sorted orders within the clusters. This way it makes it clear that the caller must ensure these properties of the input when setting this flag.

I'm just curious whether Spark does global sort on partitions keys and sorting keys or shuffles data on partition keys, then sorts on partition keys and sorting keys within each shuffle partition.

Reply:
Done.